src/containers/ServiceAccounts/ServiceAccounts.js

```diff
@@ -119,7 +118,16 @@ export /* istanbul ignore next */ class ServiceAccounts extends Component {
       id: `${serviceAccount.metadata.namespace}:${
         serviceAccount.metadata.name
       }`,
-      name: serviceAccount.metadata.name,
+      name: (
+        <Link
+          to={urls.serviceAccounts.byName({
+            namespace: serviceAccount.metadata.namespace,
+            serviceAccountName: serviceAccount.metadata.name
+          })}
+        >
```

Comment:
need to add `title={serviceAccount.metadata.name}` to the link in case long service accounts names are truncated (same already implemented on other pages)